lib/gtfs/components/gtfssearch.js

```diff
@@ -123,6 +113,26 @@ export default class GtfsSearch extends Component {
       })
   }
 
+  _routeToOption = route => (
```

Comment:
Since routeToOption and stopToOption are only used in one other place, I think they are more appropriate in the places they're called from.

Reply:
ok, good point.  I'll put them back where they were.